scripts/lib/install-node

```diff
@@ -9,6 +9,21 @@ fi
 
 source "$NVM_DIR/nvm.sh"
 node_version=6.6.0
+npm_version=3.10.3
+# if the hashed_value is not changed then exit quickly
+script_hash=`cat scripts/lib/install-node scripts/setup/node-wrapper scripts/setup/npm-wrapper`
+hashed_value=`( echo $node_version ; echo $npm_version ;  echo $script_hash) | sha1sum | awk '{ print $1 }'`
+
+d=`cat var/node_npm_version_hash`
+
+if [ "$hashed_value" == "$d" ]; then
+    echo "Node version "$node_version" is already installed."
```

Comment:
It's not that node version is correct, there's also npm and the other scripts. So this message could be more verbose.